editor/plugins/script_editor_plugin.cpp

```diff
@@ -3606,6 +3606,7 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 
 ScriptEditor::~ScriptEditor() {
 	memdelete(completion_cache);
+	memdelete(find_replace_bar);
```

Comment:
This causes a heap-use-after-free: https://github.com/godotengine/godot/pull/49650/checks?check_run_id=2837442073